google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/TableInfo.java

```diff
@@ -399,6 +412,14 @@ public BigInteger getNumRows() {
     return labels.userMap();
   }
 
+  /**
+   * If set to true, queries over this table require a partition filter (that can be used for
+   * partition elimination) to be specified.
+   */
+  public Boolean getRequirePartitionFilter() {
+    return requirePartitionFilter;
```

Comment:
The meet the javadoc style guide the comment should start with a verb.
I would rephrase:
```
/**
  * Returns true if a partition filter (that can be used for partition elimination) is required 
  * to be specified for queries over this table.
  */
```

Reply:
it's copied from TimePartitioning

Reply:
Done Thanks